mifosng-android/src/main/java/com/mifos/mifosxdroid/login/LoginActivity.java

```diff
@@ -240,16 +264,7 @@ public void onLoginClick(Button button) {
         // Saving port
         PrefManager.setPort(et_port.getEditableText().toString());
         // Updating Services
-        if (!PrefManager.getInstanceDomain().equals(BaseUrl.API_ENDPOINT)) {
-            BaseApiManager.createService();
-        }
-
-        login(false);
-    }
-
-    private void login(boolean shouldByPassSSLSecurity) {
-        if (!validateUserInputs())
-            return;
+        BaseApiManager.createService();
```

Comment:
I am also talking about recreating service. Suppose, I changed Base URL to www.chirag.com. Therefore, InstanceDomain in PrefManager will be changed to "www.chirag.com". Now, since calling on this website for api will generate an exception. Hence user will not be able to login.
Now I again changed the Base URL to the default one i,e demo.openmf.org. Now, InstanceDomain in PrefManager will be changed to "demo.openmf.org". So, now since InstanceDomain in PrefManager == BaseUrl.API_ENDPOINT , the BaseApiManager.createService() won't be called and hence user won't be able to login again.